test/API/CMakeLists.txt

```diff
@@ -309,17 +215,6 @@ if (HDF5_TEST_SERIAL)
       set (last_api_test "h5_api_test_${api_test}")
     endforeach ()
 
-    foreach (hdf5_test ${HDF5_API_TESTS_EXTRA})
```

Comment:
Only run testhdf5, etc. with VOL connectors. Don't run it twice with the native connector